schedule/yam/agama/agama_installation.yaml

```diff
@@ -1,4 +1,4 @@
----
+name: agama_installation
```

Comment:
Yes, it was indeed a mistake. I changed the name of the test in order to run VRs with different name. Sorry for not noticing it in the PR.